site/pt/_partials/devtools/whats-new.md

```diff
@@ -5,28 +5,27 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
 {# $content #}
 
 
-{# ### Chrome 111 {: #chrome111 }
+### Chrome 111 {: #chrome111 }
 
-* [Debugging HD color with the Styles pane](/pt/blog/new-in-devtools-111/#color)
-* [Enhanced breakpoint UX](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
-* [Customizable Recorder shortcuts](/pt/blog/new-in-devtools-111/#recorder)
-* [Better syntax highlight for Angular](/pt/blog/new-in-devtools-111/#syntax)
-* [Reorganize caches in the Application panel](/pt/blog/new-in-devtools-111/#cache)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-111/#misc)
- #}
+* [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)
```

Comment:
We have used "depuração" as a translation to debugging in previous articles ([example](https://github.com/GoogleChrome/developer.chrome.com/blob/10bdcea52ff4c78a301d5e56327f16a04fafa2e6/site/pt/blog/new-in-devtools-109/index.md?plain=1#L96)). I'm not picky on using anglicisms when the english equivalent is used by developers more frequently than the translated version, as I think is the case here, but would like to keep consistency across articles. Happy to standardise to one or the other. I think this is easy to get inconsistent between translators and I'm wondering if it is worth creating a page inside the [handbook](https://developer.chrome.com/docs/handbook/) to help with this. WDYT?